Deeploy/Targets/Generic/Parsers.py

```
@@ -3124,3 +3215,236 @@ class GlobalMaxPoolParser(GlobalPoolParser):
+        self.operatorRepresentation['data_in'] = data_in.name
+        self.operatorRepresentation['data_out'] = data_out.name
+        self.operatorRepresentation['batch_size'] = N
+        self.operatorRepresentation['channels'] = C
+
+        return ctxt, True
+
+
+class ResizeParser(NodeParser):
+
+    @staticmethod
+    def _is_empty(input: gs.Variable | gs.Constant | None) -> bool:
```

Ruff (0.15.20)

[error] Function argument `input` is shadowing a Python builtin

(A002)